docs/source/nlp/intro.rst

```diff
@@ -2,10 +2,26 @@ Natural Language Processing (NLP)
 =================================
 
 .. toctree::
-   :maxdepth: 8
+   :maxdepth: 2
 
    models
+   megatron_finetuning
    api
 
 
-NLP tutorials can be found under ``<NeMo_Git_root>/tutorials/nlp/``
+Scripts for running NLP models, could be found under ``NeMo/example/nlp/``.
+
+NLP tutorials are located under ``NeMo/tutorials/nlp/``.
+
+
+All examples from NLP collection can be found under `NeMo/example/nlp/ <https://github.com/NVIDIA/NeMo/tree/main/examples/nlp>`__.
+NLP notebook-tutorials are located under `NeMo/example/nlp/ <https://github.com/NVIDIA/NeMo/tree/main/tutorials/nlp>`__
+(most NeMo tutorials can be run on `Google's Colab <https://colab.research.google.com/notebooks/intro.ipynb>`_.).
+
+If you're just starting with NeMo, the best way to start is to take a look at:
+
+* `NeMo Primer <https://github.com/NVIDIA/NeMo/blob/main/tutorials/00_NeMo_Primer.ipynb>`__ - introduces NeMo, PyTorch Lightning and OmegaConf, and shows how to use, modify, save, and restore NeMo Models
+* `NeMo Models <https://github.com/NVIDIA/NeMo/blob/main/tutorials/01_NeMo_Models.ipynb>`__ - fundamental concepts of the NeMo Model
+* `NeMo voice swap demo <https://github.com/NVIDIA/NeMo/blob/main/tutorials/NeMo_voice_swap_app.ipynb>`__ - demonstrates how to swap a voice in the audio fragment with a computer generated one using NeMo.
```

Comment:
Is this tutorial useful to start with nlp? how about adding a link to a simple tutorial on a simple task here? like text classification?